src/components/TextInput/BaseTextInput.js

```diff
@@ -129,7 +129,13 @@ class BaseTextInput extends Component {
     onBlur(event) {
         if (this.props.onBlur) { this.props.onBlur(event); }
         this.setState({isFocused: false});
-        this.deactivateLabel();
+
+        // If the text has been supplied by Chrome autofill, the value state is not synced with the value
+        // as Chrome doesn't trigger a change event. When there is autofill text, don't deactivate label.
+        const textWasAutoFilled = this.input.matches && this.input.matches(':-internal-autofill-selected');
+        if (!textWasAutoFilled) {
+            this.deactivateLabel();
+        }
```

Comment:
Because we're checking the CSS selector, this PR is web specific. However, I don't know if we should create a platform specific file just because of this small change. We have to duplicate most of the code for TextInput.

Reply:
@parasharrajat please let me know what you think.

Reply:
In my understanding, `input.matches` is a method that only exists on web environments. So I think it still makes sense to create a lib method. I'll wait for @stitesExpensify's response

Reply:
@yuwenmemon @stitesExpensify is OOO. Cloud you check this one for us, please?

Reply:
I think it's fine as-is.